### ISSUE TITLE
Windows MSI built on Linux is 3× larger than if built on Windows

### ISSUE THREAD

**Issue by andersk** [electron-userland/electron-builder#5450]
* **Version**: 22.9.1
* **Electron Version**: 11.0.3
* **Electron Type (current, beta, nightly)**: current
* **Target**: Windows MSI

When I tried building [zulip-desktop](https://github.com/zulip/zulip-desktop) for Windows on the [provided Docker image](https://www.electron.build/multi-platform-build#docker), the resulting MSI was 180 MB, compared to the 60 MB I get from building it on Windows.

This can be reproduced with other apps, e.g. [electron-webpack-quick-start](https://github.com/electron-userland/electron-webpack-quick-start):

```console
$ git clone -q https://github.com/electron-userland/electron-webpack-quick-start.git
$ cd electron-webpack-quick-start
$ curl -sO https://raw.githubusercontent.com/electron/electron/v11.0.3/spec/fixtures/assets/icon.ico
$ docker run --rm -ti -v $PWD:/project electronuserland/builder:wine
root@672d26c2db30:/project# yarn add electron-build@latest
…
├─ electron-builder@22.9.1
…
root@672d26c2db30:/project# yarn dist --win msi
…
root@672d26c2db30:/project# ls -l dist
total 170392
-rw-r--r-- 1 root root       372 Dec  2 05:48  builder-effective-config.yaml
-rwxrwxr-x 1 root root 174474240 Dec  2 05:49 'electron-webpack-quick-start 0.0.0.msi'
drwxr-xr-x 1 root root        36 Dec  2 05:48  main
drwxr-xr-x 1 root root        72 Dec  2 05:48  renderer
drwxr-xr-x 1 root root       600 Dec  2 05:48  win-unpacked
```

Building on this Windows instead gives a 57520128 byte MSI. The `win-unpacked` folder is the same size, so I suspect a compression issue.

**Comment by andersk** [comment]
Yes, this still reproduces with the latest version of electron-builder. (Which is the same as the latest version of electron-builder when I originally reported the bug. Maybe this stalebot configuration is a bit aggressive?)

**Comment by kolaente** [comment]
Yes, still relevant with version 22.11.5

**Comment by kolaente** [comment]
Yes, still relevant with the latest version.

**Comment by andersk** [comment]
Still an issue in 23.0.3.

**Comment by FizzBuzz791** [comment]
It looks like compression doesn't work if you're not on a native windows OS. The `win-upacked` dir is ~500Mb and the .msi is slightly higher than that (which makes sense).

On a native windows OS, the `win-unpacked` dir is ~500Mb but then the .msi is only 151Mb, which is much closer to what I'd expect.
